e2e/setup/config.go

```diff
@@ -133,16 +141,21 @@ func newSetup(config Config) (*Setup, error) {
 		return nil, err
 	}
 
+	dbValidator := db_validation.NewService(config.Hedera.DbValidationProps)
+
 	return &Setup{
-		BridgeAccount: bridgeAccount,
-		SenderAccount: senderAccount,
-		TopicID:       topicID,
-		TokenID:       tokenID,
-		FeePercentage: config.Hedera.FeePercentage,
-		Members:       members,
-		Clients:       clients,
-		DbValidation:  db_validation.NewService(config.Hedera.DbValidationProps),
-		Receiver:      common.HexToAddress(clients.Signer.Address()),
+		BridgeAccount:             bridgeAccount,
+		HederaSender:              senderAccount,
+		HederaReceiver:            receiverAccount,
+		EthSender:                 config.Ethereum.Sender,
+		EthReceiver:               common.HexToAddress(config.Ethereum.Receiver.Address),
```

Comment:
To not introduce duplicate fields, you can use the signer's address as previously.